src/Brick/Widgets/List.hs

```diff
@@ -98,6 +99,30 @@ handleListEvent e theList =
         EvKey KPageUp [] -> listMovePageUp theList
         _ -> return theList
 
+-- | Enable list movement with the vi keys with a fallback if none match.
+-- Movements handled: Up, Down, Page Up, Page Down, Half Page Up, Half Page
```

Comment:
Please document which Vi keys correspond with which behaviors in this Haddock.

Reply:
I was on the fence about this since these are all standard bindings, but I can document them.

Reply:
Thanks! Even if they are standard bindings, documenting them makes this handler consistent with the other list documentation and makes this library easier to use for someone who is *not* already a `vi` user.